build/local_settings.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-DEBUG = True
+DEBUG = False

```

Comment:
Keep these true

Reply:
I didn't change these, I think you did, but I'll fix it now

Reply:
I'll screenshot, but I don't think you have to rebuild the container?